File: app/app/Model/Member/Response/MemberGetResponse.swift
struct MemberGetResponse: Codable, Identifiable {

    let memberId: Int64
    let images: [MemberPublicImageResponse]
    let nickname: String
    let gender: String
    let age: Int
    var likes: Int64
    let distance: Double?
    let bio: String?
    let updatedAt: String
    let isChatEnabled: Bool
    var isLike: Bool
    let isPrivatePhoto: Bool
    var isOpenPrivatePhoto: Bool
    var isBlock: Bool

    var id: Int64 { memberId }
}

struct MemberPublicImageResponse: Codable {

    let url: String
}

File: server/src/main/kotlin/com/openmpy/server/member/dto/response/MemberGetResponse.kt
package com.openmpy.server.member.dto.response

import com.openmpy.server.member.domain.type.Gender
import java.time.LocalDateTime

data class MemberGetResponse(

    val memberId: Long,
    val images: List<MemberPublicImageResponse>,
    val nickname: String,
    val gender: Gender,
    val age: Int,
    val likes: Long,
    val distance: Double?,
    val bio: String?,
    val isChatEnabled: Boolean,
    val updatedAt: LocalDateTime,
    val isLike: Boolean,
    val isPrivatePhoto: Boolean,
    val isOpenPrivatePhoto: Boolean,
    val isBlock: Boolean,
)

data class MemberPublicImageResponse(

    val url: String,
)
